scripts/dangerously-pull-training-repo.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e # exit if any command fails
+
+
+BRANCHES="react-fundamentals graphql-fundamentals advanced-graphql"
+
+echo "⚠️ WARNING! This will do a HARD RESET and wipe out your history on:"
+for BRANCH in $BRANCHES ; do
+  echo "🔥 $BRANCH"
+done
+
+echo "Abort now if you haven't pushed all of those branches!"
+echo "(Or get to know git reflog...)"
+echo "Press enter to continue."
+read || exit
+
+if [ ! -z "$(git status --porcelain)" ]; then
```

Comment:
I'm wondering if this should go above line 13 above (same for the other script). I'd think we don't need to ask for confirmation if the script is going to fail.

Reply:
Good point, done.